filament/src/components/TransformManager.h

```diff
@@ -234,8 +234,8 @@ class UTILS_PRIVATE FTransformManager : public TransformManager {
                 Field<WORLD>        world;
                 Field<LOCAL_LO>     localTranslationLo;
                 Field<WORLD_LO>     worldTranslationLo;
-                Field<MATERIAL_LOCAL_ORIENTATION>           materialLocalOrientation;
-                Field<MATERIAL_ORIENTATION>                 materialOrientation;
+                Field<MATERIAL_LOCAL_ORIENTATION>      materialLocalOrientation;
+                Field<MATERIAL_ORIENTATION>            materialOrientation;
```

Comment:
nit: lets not change the indentation
```suggestion
                Field<MATERIAL_LOCAL_ORIENTATION>           materialLocalOrientation;
                Field<MATERIAL_ORIENTATION>                 materialOrientation;
```

Reply:
https://github.com/shapr3d/filament/pull/162/commits/6b05fb0008899ce758a2cd8cfc8e1e4b2f094c46